config/routes.rb

```diff
@@ -5,9 +5,9 @@
   resources :question_inputs
   resources :questions
   resources :question_types
-  resources :content_resources
-  resources :content_metadata
-  resources :content_audios
+  resources :content_resources, except: %i[update show index]
+  resources :content_metadata, except: %i[update show index]
+  resources :content_audios, except: %i[update show index]
```

Comment:
Let's keep it in mind that we might want to prevent deletes as well. Not really sure yet. I user might accidentally add a resource with incorrect content_id and thus it should be possible to delete it. At the same time I'd like to prevent accidental deletes of resources.

Reply:
I'll leave deleting in for now, but perhaps I could add a comment that delete should be prohibited at some later point?